src/components/SideBar.tsx

```diff
@@ -16,7 +16,7 @@ import { createPortal } from 'react-dom';
 import StyledInput from './StyledInput';
 
 
-export default function SideBar() {
+export default function SideBar({isChatPage = false}: { isChatPage?: boolean }) {
```

Comment:
- [ ] SideBar - is a common component and shouldn't know about details of some particular page. It should consist of a theme switcher and open/close - left panel